consensus/service/src/peer_api_service.rs

```diff
@@ -281,6 +275,24 @@ impl<E: ConsensusEnclave, L: Ledger, TXM: TxManager + Clone> ConsensusPeerApi
                 return;
             }
 
+            let unverified_consensus_msg: mc_peers::ConsensusMsg =
```

Comment:
I moved this down so that the node doesn't bother deserializing a message from an unknown peer.